### PR TITLE
Add test cases for pushdown utils

### PR DESCRIPTION
**Pull request by alexr-bq**
### Summary

Add test cases for pushdown utils.

### Related Issue

VER-76316
 
### Additional Reviewers

@jonathanl-bq 
@ravjotbrar

### CODE REVIEW

File: connector/src/main/scala/com/vertica/spark/util/pushdown/PushdownUtils.scala [hunk 1]
```diff
@@ -1,36 +1,7 @@
-// (c) Copyright [2020-2021] Micro Focus or one of its affiliates.
```
Comment (jonathanl-bq) [inline connector/src/main/scala/com/vertica/spark/util/pushdown/PushdownUtils.scala:1]: How did the static analysis phase pass without this header?

Comment (jonathanl-bq) [reply connector/src/main/scala/com/vertica/spark/util/pushdown/PushdownUtils.scala:1]: Oh nevermind, I see it didn't. Update this and you're good.

File: connector/src/test/scala/com/vertica/spark/util/pushdown/PushdownUtilsTest.scala [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+import org.apache.spark.sql.sources._
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.flatspec.AnyFlatSpec
+
+class PushdownUtilsTest extends AnyFlatSpec with BeforeAndAfterAll with MockFactory with org.scalatest.OneInstancePerTest {
+
+  val sparkFilters: Array[Filter] = Array(
+    EqualTo("a", 5),
```
Comment (alexey-temnikov) [inline connector/src/test/scala/com/vertica/spark/util/pushdown/PushdownUtilsTest.scala:24]: Should we list in documentation what filters we are supporting, as well as list all Basic data types which are tested / supported?

File: connector/src/main/scala/com/vertica/spark/util/pushdown/PushdownUtils.scala (hunk 1 above)
Comment (alexr-bq) [reply connector/src/main/scala/com/vertica/spark/util/pushdown/PushdownUtils.scala:1]: Don't know how this was removed in the first place, I think intellij did it.

**Review by jonathanl-bq — APPROVED** [review]

**Pull request merged**